src/wagtail_ai/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+from .prompts import Prompt as PromptDataClass
+
+
+class Prompt(models.Model):
+    label = models.CharField(max_length=255)
```

Comment:
**Question:** The label is displayed in the UI, right? Would it ever need to be this long?